torch/testing/_internal/distributed/nn/api/remote_module_test.py

```diff
@@ -208,6 +211,25 @@ def test_remote_parameters(self):
             self.assertEqual(len(param_rrefs), 1)
             self.assertTrue(torch.equal(param_rrefs[0].to_here(), _PARAM_VAL))
 
+    @dist_utils.dist_init
+    def test_device(self):
```

Comment:
Can we add a test for invalid devices (ex: "foo", "cuda:100", "cpu2" etc.) and validate that we raise appropriate exceptions.

Reply:
Added another test_invalid_device testcase.